index.html

```diff
@@ -1,184 +1,288 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>Microverse Portfolio</title>
-</head>
-<body>
+  </head>
+  <body>
     <header>
-        <nav>
-            <a href="#">My Logo</a>
-            <img src="./assets/menu.png" alt="logo">
-        </nav>
-        <div class="background">
-            <div class="intro">
-                <div class="text">
-                    <h1>I'm Favour. <br> Glad to see you!</h1>
-                </div>
-                <p class="intro-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-                
-                <div class="socials">
```

Comment:
Well done  improving the style of your project 👏 as requested by the previous CR .  . However, there are still little things that will make your project even better.

* [ ]  The left and right padding you have on the navbar and about section are quite very small compared to the `155px` as provided in the Figma Design. Kindly increase the padding on the left and right to about `155px`.

Figma design ![image](https://user-images.githubusercontent.com/84156714/236190544-9965bbe6-652b-41e0-ad86-2ccc7dcd2189.png)

Your design ![image](https://user-images.githubusercontent.com/84156714/236191764-17d49f08-88ff-45a7-95a7-4b7fe6e63796.png)

I use the picture dropped by the previous reviewer because they are descriptive , fix your design accordingly 👍

Reply:
Please I updated the style and everything was ok when I changed it. Please can you re-check it again?